[2023-11-16T11:43:56.662+0000] {processor.py:157} INFO - Started process (PID=30) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:43:56.666+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:43:56.697+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:43:56.696+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:43:56.725+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:43:57.009+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:43:57.009+0000] {manager.py:499} INFO - Created Permission View: %s
[2023-11-16T11:43:57.015+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:43:57.015+0000] {manager.py:499} INFO - Created Permission View: %s
[2023-11-16T11:43:57.019+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:43:57.018+0000] {manager.py:499} INFO - Created Permission View: %s
[2023-11-16T11:43:57.019+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:43:57.019+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:43:57.026+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:43:57.026+0000] {dag.py:2963} INFO - Creating ORM DAG for Backend_run
[2023-11-16T11:43:57.033+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:43:57.033+0000] {dag.py:3722} INFO - Setting next_dagrun for Backend_run to 2023-01-01T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2023-11-16T11:43:57.042+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.385 seconds
[2023-11-16T11:44:27.334+0000] {processor.py:157} INFO - Started process (PID=38) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:44:27.337+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:44:27.348+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:44:27.346+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:44:27.424+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:44:27.503+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:44:27.502+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:44:27.560+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:44:27.560+0000] {dag.py:3722} INFO - Setting next_dagrun for Backend_run to 2023-01-01T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2023-11-16T11:44:27.578+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.255 seconds
[2023-11-16T11:44:57.673+0000] {processor.py:157} INFO - Started process (PID=46) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:44:57.674+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:44:57.679+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:44:57.678+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:44:57.704+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:44:57.750+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:44:57.750+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:44:57.770+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:44:57.770+0000] {dag.py:3722} INFO - Setting next_dagrun for Backend_run to 2023-01-01T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2023-11-16T11:44:57.780+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.114 seconds
[2023-11-16T11:45:28.232+0000] {processor.py:157} INFO - Started process (PID=54) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:45:28.233+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:45:28.237+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:45:28.236+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:45:28.258+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:45:28.290+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:45:28.290+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:45:28.309+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:45:28.309+0000] {dag.py:3722} INFO - Setting next_dagrun for Backend_run to 2023-01-01T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2023-11-16T11:45:28.318+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.090 seconds
[2023-11-16T11:45:58.660+0000] {processor.py:157} INFO - Started process (PID=62) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:45:58.662+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:45:58.668+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:45:58.666+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:45:58.719+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:45:58.758+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:45:58.758+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:45:58.783+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:45:58.782+0000] {dag.py:3722} INFO - Setting next_dagrun for Backend_run to 2023-01-01T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2023-11-16T11:45:58.798+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.147 seconds
[2023-11-16T11:46:29.015+0000] {processor.py:157} INFO - Started process (PID=70) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:46:29.016+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:46:29.027+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:46:29.025+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:46:29.103+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:46:29.166+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:46:29.166+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:46:29.229+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:46:29.229+0000] {dag.py:3722} INFO - Setting next_dagrun for Backend_run to 2023-01-01T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2023-11-16T11:46:29.247+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.240 seconds
[2023-11-16T11:46:59.363+0000] {processor.py:157} INFO - Started process (PID=77) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:46:59.364+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:46:59.368+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:46:59.367+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:46:59.395+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:46:59.475+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:46:59.475+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:46:59.499+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:46:59.498+0000] {dag.py:3722} INFO - Setting next_dagrun for Backend_run to 2023-01-01T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2023-11-16T11:46:59.536+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.177 seconds
[2023-11-16T11:47:29.879+0000] {processor.py:157} INFO - Started process (PID=85) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:47:29.880+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:47:29.886+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:47:29.886+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:47:29.911+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:47:29.943+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:47:29.943+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:47:29.963+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:47:29.963+0000] {dag.py:3722} INFO - Setting next_dagrun for Backend_run to 2023-01-01T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2023-11-16T11:47:29.979+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.110 seconds
[2023-11-16T11:48:01.775+0000] {processor.py:157} INFO - Started process (PID=97) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:48:01.787+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:48:02.909+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:48:02.411+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:48:06.432+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:49:26.185+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:49:25.988+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:50:39.329+0000] {processor.py:157} INFO - Started process (PID=105) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:50:39.332+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:50:39.349+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:50:39.336+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:50:39.956+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:51:04.737+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:51:04.719+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:51:05.213+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 25.951 seconds
[2023-11-16T11:51:45.871+0000] {processor.py:157} INFO - Started process (PID=30) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:51:45.873+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:51:45.875+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:51:45.875+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:51:45.897+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:51:45.935+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:51:45.934+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:51:45.950+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:51:45.949+0000] {dag.py:3722} INFO - Setting next_dagrun for Backend_run to 2023-01-17T00:00:00+00:00, run_after=2023-01-18T00:00:00+00:00
[2023-11-16T11:51:45.961+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.092 seconds
[2023-11-16T11:52:16.264+0000] {processor.py:157} INFO - Started process (PID=38) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:52:16.267+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:52:16.272+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:52:16.271+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:52:16.335+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:52:16.396+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:52:16.396+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:52:16.457+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.212 seconds
[2023-11-16T11:52:46.494+0000] {processor.py:157} INFO - Started process (PID=46) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:52:46.495+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:52:46.498+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:52:46.498+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:52:46.517+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:52:46.544+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:52:46.543+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:52:46.570+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.081 seconds
[2023-11-16T11:53:16.861+0000] {processor.py:157} INFO - Started process (PID=58) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:53:16.863+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:53:16.868+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:53:16.867+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:53:16.896+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:53:16.935+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:53:16.935+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:53:16.970+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.116 seconds
[2023-11-16T11:53:47.145+0000] {processor.py:157} INFO - Started process (PID=66) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:53:47.147+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:53:47.153+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:53:47.152+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:53:47.184+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:53:47.260+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:53:47.260+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:53:47.435+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.298 seconds
[2023-11-16T11:54:17.533+0000] {processor.py:157} INFO - Started process (PID=74) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:54:17.537+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:54:17.545+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:54:17.543+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:54:17.600+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:54:17.660+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:54:17.660+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:54:17.711+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.194 seconds
[2023-11-16T11:54:48.023+0000] {processor.py:157} INFO - Started process (PID=83) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:54:48.025+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:54:48.030+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:54:48.030+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:54:48.065+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:54:48.106+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:54:48.106+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:54:48.168+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.154 seconds
[2023-11-16T11:55:18.432+0000] {processor.py:157} INFO - Started process (PID=91) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:55:18.438+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:55:18.456+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:55:18.453+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:55:18.489+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:55:18.522+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:55:18.522+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:55:18.555+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.141 seconds
[2023-11-16T11:55:48.980+0000] {processor.py:157} INFO - Started process (PID=99) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:55:48.982+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:55:48.987+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:55:48.986+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:55:49.018+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:55:49.052+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:55:49.052+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:55:49.082+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.109 seconds
[2023-11-16T11:56:19.284+0000] {processor.py:157} INFO - Started process (PID=107) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:56:19.286+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:56:19.291+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:56:19.290+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:56:19.332+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:56:19.375+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:56:19.375+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:56:19.411+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.149 seconds
[2023-11-16T11:56:49.653+0000] {processor.py:157} INFO - Started process (PID=115) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:56:49.655+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:56:49.661+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:56:49.660+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:56:49.689+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:56:49.741+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:56:49.741+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:56:49.777+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.131 seconds
[2023-11-16T11:57:20.078+0000] {processor.py:157} INFO - Started process (PID=123) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:57:20.079+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:57:20.083+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:57:20.082+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:57:20.107+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:57:20.144+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:57:20.144+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:57:20.189+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.136 seconds
[2023-11-16T11:57:50.444+0000] {processor.py:157} INFO - Started process (PID=131) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:57:50.446+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:57:50.449+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:57:50.449+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:57:50.477+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:57:50.527+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:57:50.527+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:57:50.556+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.118 seconds
[2023-11-16T11:58:20.831+0000] {processor.py:157} INFO - Started process (PID=139) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:58:20.833+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:58:20.838+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:58:20.837+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:58:20.864+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:58:20.905+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:58:20.905+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:58:20.932+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.109 seconds
[2023-11-16T11:58:51.166+0000] {processor.py:157} INFO - Started process (PID=147) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:58:51.168+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:58:51.171+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:58:51.171+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:58:51.200+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:58:51.230+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:58:51.230+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:58:51.255+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.098 seconds
[2023-11-16T11:59:21.535+0000] {processor.py:157} INFO - Started process (PID=155) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:59:21.537+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:59:21.541+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:59:21.540+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:59:21.571+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:59:21.622+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:59:21.622+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:59:21.648+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.120 seconds
[2023-11-16T11:59:51.880+0000] {processor.py:157} INFO - Started process (PID=163) to work on /opt/airflow/dags/dags.py
[2023-11-16T11:59:51.882+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T11:59:51.886+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:59:51.885+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T11:59:51.920+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T11:59:51.971+0000] {logging_mixin.py:154} INFO - [2023-11-16T11:59:51.971+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T11:59:52.003+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.129 seconds
[2023-11-16T12:00:22.257+0000] {processor.py:157} INFO - Started process (PID=171) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:00:22.258+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:00:22.263+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:00:22.261+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:00:22.302+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:00:22.348+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:00:22.348+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:00:22.377+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.126 seconds
[2023-11-16T12:00:52.509+0000] {processor.py:157} INFO - Started process (PID=178) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:00:52.510+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:00:52.514+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:00:52.513+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:00:52.533+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:00:52.566+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:00:52.566+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:00:52.591+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.086 seconds
[2023-11-16T12:01:22.812+0000] {processor.py:157} INFO - Started process (PID=185) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:01:22.813+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:01:22.817+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:01:22.816+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:01:22.836+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:01:22.866+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:01:22.866+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:01:22.888+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.081 seconds
[2023-11-16T12:01:53.202+0000] {processor.py:157} INFO - Started process (PID=193) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:01:53.204+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:01:53.208+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:01:53.207+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:01:53.227+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:01:53.257+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:01:53.256+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:01:53.278+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.081 seconds
[2023-11-16T12:02:23.556+0000] {processor.py:157} INFO - Started process (PID=201) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:02:23.558+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:02:23.563+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:02:23.562+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:02:23.588+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:02:23.632+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:02:23.632+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:02:23.659+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.117 seconds
[2023-11-16T12:02:53.806+0000] {processor.py:157} INFO - Started process (PID=209) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:02:53.808+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:02:53.815+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:02:53.814+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:02:53.843+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:02:53.883+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:02:53.882+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:02:53.911+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.111 seconds
[2023-11-16T12:03:24.149+0000] {processor.py:157} INFO - Started process (PID=217) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:03:24.150+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:03:24.154+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:03:24.153+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:03:24.172+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:03:24.205+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:03:24.205+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:03:24.238+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.100 seconds
[2023-11-16T12:03:54.439+0000] {processor.py:157} INFO - Started process (PID=225) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:03:54.441+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:03:54.445+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:03:54.445+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:03:54.476+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:03:54.513+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:03:54.513+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:03:54.539+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.106 seconds
[2023-11-16T12:04:24.772+0000] {processor.py:157} INFO - Started process (PID=233) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:04:24.773+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:04:24.777+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:04:24.776+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:04:24.811+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:04:24.842+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:04:24.842+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:04:24.868+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.101 seconds
[2023-11-16T12:04:55.119+0000] {processor.py:157} INFO - Started process (PID=241) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:04:55.120+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:04:55.124+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:04:55.124+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:04:55.148+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:04:55.186+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:04:55.186+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:04:55.214+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.102 seconds
[2023-11-16T12:05:25.425+0000] {processor.py:157} INFO - Started process (PID=249) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:05:25.427+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:05:25.441+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:05:25.440+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:05:25.484+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:05:25.512+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:05:25.512+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:05:25.536+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.119 seconds
[2023-11-16T12:05:55.733+0000] {processor.py:157} INFO - Started process (PID=257) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:05:55.734+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:05:55.737+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:05:55.736+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:05:55.754+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:05:55.783+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:05:55.783+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:05:55.806+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.077 seconds
[2023-11-16T12:06:26.071+0000] {processor.py:157} INFO - Started process (PID=265) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:06:26.072+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:06:26.076+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:06:26.075+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:06:26.137+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:06:26.178+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:06:26.178+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:06:26.204+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.138 seconds
[2023-11-16T12:06:56.556+0000] {processor.py:157} INFO - Started process (PID=272) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:06:56.557+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:06:56.562+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:06:56.561+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:06:56.590+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:06:56.628+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:06:56.628+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:06:56.671+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.122 seconds
[2023-11-16T12:07:26.880+0000] {processor.py:157} INFO - Started process (PID=280) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:07:26.881+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:07:26.886+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:07:26.885+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:07:26.910+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:07:26.939+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:07:26.938+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:07:26.961+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.089 seconds
[2023-11-16T12:07:57.191+0000] {processor.py:157} INFO - Started process (PID=288) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:07:57.192+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:07:57.196+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:07:57.196+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:07:57.221+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:07:57.248+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:07:57.248+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:07:57.270+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.086 seconds
[2023-11-16T12:08:27.550+0000] {processor.py:157} INFO - Started process (PID=297) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:08:27.552+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:08:27.557+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:08:27.556+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:08:27.604+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:08:27.633+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:08:27.633+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:08:27.657+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.114 seconds
[2023-11-16T12:08:57.857+0000] {processor.py:157} INFO - Started process (PID=305) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:08:57.858+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:08:57.863+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:08:57.862+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:08:57.907+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:08:57.941+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:08:57.941+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:08:57.966+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.117 seconds
[2023-11-16T12:09:28.179+0000] {processor.py:157} INFO - Started process (PID=313) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:09:28.181+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:09:28.187+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:09:28.186+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:09:28.216+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:09:28.251+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:09:28.251+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:09:28.285+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.115 seconds
[2023-11-16T12:09:58.533+0000] {processor.py:157} INFO - Started process (PID=321) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:09:58.535+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:09:58.541+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:09:58.540+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:09:58.568+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:09:58.598+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:09:58.598+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:09:58.623+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.099 seconds
[2023-11-16T12:10:28.860+0000] {processor.py:157} INFO - Started process (PID=329) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:10:28.862+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:10:28.868+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:10:28.867+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:10:28.896+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:10:28.931+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:10:28.931+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:10:28.968+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.117 seconds
[2023-11-16T12:10:59.174+0000] {processor.py:157} INFO - Started process (PID=337) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:10:59.175+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:10:59.178+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:10:59.177+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:10:59.197+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:10:59.233+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:10:59.233+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:10:59.253+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.088 seconds
[2023-11-16T12:11:29.506+0000] {processor.py:157} INFO - Started process (PID=345) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:11:29.507+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:11:29.513+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:11:29.512+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:11:29.539+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:11:29.567+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:11:29.567+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:11:29.587+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.090 seconds
[2023-11-16T12:11:59.884+0000] {processor.py:157} INFO - Started process (PID=353) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:11:59.886+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:11:59.890+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:11:59.888+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:11:59.911+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:11:59.936+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:11:59.936+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:11:59.954+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.075 seconds
[2023-11-16T12:12:30.276+0000] {processor.py:157} INFO - Started process (PID=361) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:12:30.277+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:12:30.282+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:12:30.281+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:12:30.307+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:12:30.337+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:12:30.337+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:12:30.358+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.091 seconds
[2023-11-16T12:13:00.576+0000] {processor.py:157} INFO - Started process (PID=369) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:13:00.576+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:13:00.580+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:13:00.579+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:13:00.598+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:13:00.625+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:13:00.625+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:13:00.648+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.076 seconds
[2023-11-16T12:13:30.896+0000] {processor.py:157} INFO - Started process (PID=377) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:13:30.898+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:13:30.907+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:13:30.905+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:13:30.958+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:13:31.110+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:13:31.108+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:13:31.143+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.254 seconds
[2023-11-16T12:13:48.319+0000] {processor.py:157} INFO - Started process (PID=385) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:13:48.321+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:13:48.325+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:13:48.325+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:13:48.335+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:13:48.331+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 2, in <module>
    from airflow.operators.bash_operator import DockerOperator
ImportError: cannot import name 'DockerOperator' from 'airflow.operators.bash_operator' (unknown location)
[2023-11-16T12:13:48.335+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:13:48.360+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.047 seconds
[2023-11-16T12:14:18.692+0000] {processor.py:157} INFO - Started process (PID=393) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:14:18.695+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:14:18.706+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:14:18.701+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:14:18.741+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:14:18.729+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 2, in <module>
    from airflow.operators.bash_operator import DockerOperator
ImportError: cannot import name 'DockerOperator' from 'airflow.operators.bash_operator' (unknown location)
[2023-11-16T12:14:18.742+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:14:18.788+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.105 seconds
[2023-11-16T12:15:01.957+0000] {processor.py:157} INFO - Started process (PID=401) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:15:05.392+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:15:08.179+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:15:07.732+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:15:10.174+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:15:09.576+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 2, in <module>
    from airflow.operators.bash_operator import DockerOperator
ImportError: cannot import name 'DockerOperator' from 'airflow.operators.bash_operator' (unknown location)
[2023-11-16T12:15:10.681+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:15:28.186+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 30.386 seconds
[2023-11-16T12:16:05.033+0000] {processor.py:157} INFO - Started process (PID=409) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:16:05.064+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:16:05.330+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:16:05.264+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:16:06.479+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:16:06.174+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 2, in <module>
    from airflow.operators.bash_operator import DockerOperator
ImportError: cannot import name 'DockerOperator' from 'airflow.operators.bash_operator' (unknown location)
[2023-11-16T12:16:06.501+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:16:35.276+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 30.273 seconds
[2023-11-16T12:17:13.648+0000] {processor.py:157} INFO - Started process (PID=423) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:17:13.653+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:17:13.664+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:17:13.659+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:17:13.713+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:17:13.702+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 2, in <module>
    from airflow.operators.bash_operator import DockerOperator
ImportError: cannot import name 'DockerOperator' from 'airflow.operators.bash_operator' (unknown location)
[2023-11-16T12:17:13.714+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:17:13.794+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.206 seconds
[2023-11-16T12:17:43.834+0000] {processor.py:157} INFO - Started process (PID=432) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:17:43.835+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:17:43.836+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:17:43.836+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:17:43.844+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:17:43.841+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 2, in <module>
    from airflow.operators.bash_operator import DockerOperator
ImportError: cannot import name 'DockerOperator' from 'airflow.operators.bash_operator' (unknown location)
[2023-11-16T12:17:43.844+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:17:43.878+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.050 seconds
[2023-11-16T12:18:16.635+0000] {processor.py:157} INFO - Started process (PID=433) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:16.646+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:16.655+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:16.652+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:16.721+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:16.697+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 2, in <module>
    from airflow.operators.bash_operator import DockerOperator
ImportError: cannot import name 'DockerOperator' from 'airflow.operators.bash_operator' (unknown location)
[2023-11-16T12:18:16.721+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:24.202+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 7.665 seconds
[2023-11-16T12:18:42.479+0000] {processor.py:157} INFO - Started process (PID=442) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.484+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:42.490+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.487+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.524+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.516+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:42.525+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.586+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.141 seconds
[2023-11-16T12:18:42.602+0000] {processor.py:157} INFO - Started process (PID=443) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.603+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:42.604+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.603+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.607+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.606+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:42.608+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.620+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.020 seconds
[2023-11-16T12:18:42.633+0000] {processor.py:157} INFO - Started process (PID=444) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.634+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:42.634+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.634+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.637+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.636+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:42.637+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.648+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.017 seconds
[2023-11-16T12:18:42.661+0000] {processor.py:157} INFO - Started process (PID=445) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.662+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:42.662+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.662+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.666+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.665+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:42.666+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.676+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.018 seconds
[2023-11-16T12:18:42.688+0000] {processor.py:157} INFO - Started process (PID=446) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.689+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:42.689+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.689+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.692+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.691+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:42.692+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.701+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:42.715+0000] {processor.py:157} INFO - Started process (PID=447) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.716+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:42.716+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.716+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.719+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.718+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:42.719+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.729+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:42.741+0000] {processor.py:157} INFO - Started process (PID=448) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.742+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:42.743+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.742+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.745+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.745+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:42.746+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.758+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.020 seconds
[2023-11-16T12:18:42.775+0000] {processor.py:157} INFO - Started process (PID=449) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.776+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:42.777+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.777+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.780+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.779+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:42.780+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.791+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.019 seconds
[2023-11-16T12:18:42.836+0000] {processor.py:157} INFO - Started process (PID=450) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.837+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:42.839+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.838+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.857+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.853+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:42.860+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.903+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.084 seconds
[2023-11-16T12:18:42.920+0000] {processor.py:157} INFO - Started process (PID=451) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.921+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:42.921+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.921+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.924+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.924+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:42.925+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.935+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.018 seconds
[2023-11-16T12:18:42.948+0000] {processor.py:157} INFO - Started process (PID=452) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.949+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:42.949+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.949+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.952+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.952+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:42.952+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.962+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:42.975+0000] {processor.py:157} INFO - Started process (PID=453) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.976+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:42.977+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.976+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.979+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:42.979+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:42.980+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:42.990+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.017 seconds
[2023-11-16T12:18:43.003+0000] {processor.py:157} INFO - Started process (PID=454) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.004+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.005+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.004+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.007+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.007+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.007+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.018+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.017 seconds
[2023-11-16T12:18:43.030+0000] {processor.py:157} INFO - Started process (PID=455) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.031+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.032+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.031+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.034+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.034+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.034+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.047+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.018 seconds
[2023-11-16T12:18:43.060+0000] {processor.py:157} INFO - Started process (PID=456) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.061+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.061+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.061+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.064+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.064+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.065+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.074+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.017 seconds
[2023-11-16T12:18:43.087+0000] {processor.py:157} INFO - Started process (PID=457) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.088+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.088+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.088+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.091+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.090+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.091+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.100+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:43.112+0000] {processor.py:157} INFO - Started process (PID=458) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.113+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.113+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.113+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.116+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.116+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.116+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.126+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:43.141+0000] {processor.py:157} INFO - Started process (PID=459) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.142+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.143+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.142+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.146+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.145+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.146+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.155+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.017 seconds
[2023-11-16T12:18:43.168+0000] {processor.py:157} INFO - Started process (PID=460) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.168+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.169+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.168+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.171+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.171+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.172+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.181+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:43.193+0000] {processor.py:157} INFO - Started process (PID=461) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.194+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.194+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.194+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.197+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.197+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.197+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.207+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:43.220+0000] {processor.py:157} INFO - Started process (PID=462) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.221+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.221+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.221+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.224+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.223+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.224+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.234+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:43.246+0000] {processor.py:157} INFO - Started process (PID=463) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.246+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.247+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.247+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.250+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.250+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.250+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.260+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:43.272+0000] {processor.py:157} INFO - Started process (PID=464) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.272+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.273+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.273+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.276+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.275+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.276+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.285+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.015 seconds
[2023-11-16T12:18:43.297+0000] {processor.py:157} INFO - Started process (PID=465) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.297+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.298+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.298+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.301+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.300+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.301+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.311+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:43.325+0000] {processor.py:157} INFO - Started process (PID=466) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.326+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.326+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.326+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.329+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.328+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.329+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.338+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:43.351+0000] {processor.py:157} INFO - Started process (PID=467) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.351+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.352+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.351+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.355+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.354+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.355+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.364+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:43.376+0000] {processor.py:157} INFO - Started process (PID=468) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.377+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.377+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.377+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.381+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.380+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.381+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.390+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:43.403+0000] {processor.py:157} INFO - Started process (PID=469) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.404+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.405+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.404+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.407+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.407+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.407+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.417+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:18:43.435+0000] {processor.py:157} INFO - Started process (PID=470) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.436+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.437+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.437+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.442+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.441+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.442+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.453+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.024 seconds
[2023-11-16T12:18:43.467+0000] {processor.py:157} INFO - Started process (PID=471) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.468+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:18:43.468+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.468+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.471+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:18:43.470+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:18:43.471+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:18:43.481+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.016 seconds
[2023-11-16T12:19:14.154+0000] {processor.py:157} INFO - Started process (PID=479) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:19:14.156+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:19:14.157+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:19:14.156+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:19:14.169+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:19:14.165+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:19:14.170+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:19:14.208+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.066 seconds
[2023-11-16T12:19:44.513+0000] {processor.py:157} INFO - Started process (PID=487) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:19:44.515+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:19:44.518+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:19:44.516+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:19:44.527+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:19:44.524+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:19:44.527+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:19:44.557+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.056 seconds
[2023-11-16T12:20:14.817+0000] {processor.py:157} INFO - Started process (PID=495) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:20:14.819+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:20:14.821+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:20:14.820+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:20:14.831+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:20:14.829+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:20:14.831+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:20:14.854+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.046 seconds
[2023-11-16T12:20:45.180+0000] {processor.py:157} INFO - Started process (PID=503) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:20:45.182+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:20:45.184+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:20:45.182+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:20:45.193+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:20:45.190+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:20:45.193+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:20:45.214+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.041 seconds
[2023-11-16T12:21:15.564+0000] {processor.py:157} INFO - Started process (PID=511) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:21:15.566+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:21:15.567+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:21:15.567+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:21:15.577+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:21:15.575+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:21:15.577+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:21:15.602+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.047 seconds
[2023-11-16T12:21:45.912+0000] {processor.py:157} INFO - Started process (PID=519) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:21:45.914+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:21:45.916+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:21:45.915+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:21:45.924+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:21:45.922+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:21:45.925+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:21:45.945+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.041 seconds
[2023-11-16T12:22:16.307+0000] {processor.py:157} INFO - Started process (PID=527) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:22:16.309+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:22:16.310+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:22:16.310+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:22:16.319+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:22:16.317+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:22:16.320+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:22:16.343+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.042 seconds
[2023-11-16T12:22:36.607+0000] {processor.py:157} INFO - Started process (PID=535) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:22:36.609+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:22:36.612+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:22:36.611+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:22:36.619+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:22:36.617+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:22:36.619+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:22:36.642+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.043 seconds
[2023-11-16T12:23:07.002+0000] {processor.py:157} INFO - Started process (PID=544) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:23:07.004+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:23:07.007+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:23:07.004+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:23:07.016+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:23:07.012+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:23:07.016+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:23:07.044+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.050 seconds
[2023-11-16T12:23:25.288+0000] {processor.py:157} INFO - Started process (PID=552) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:23:25.289+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:23:25.291+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:23:25.290+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:23:25.308+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:23:25.300+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:23:25.309+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:23:25.353+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.074 seconds
[2023-11-16T12:23:55.654+0000] {processor.py:157} INFO - Started process (PID=560) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:23:55.656+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:23:55.658+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:23:55.657+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:23:55.669+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:23:55.666+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:23:55.670+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:23:55.696+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.051 seconds
[2023-11-16T12:24:26.026+0000] {processor.py:157} INFO - Started process (PID=568) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:24:26.028+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:24:26.030+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:24:26.029+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:24:26.040+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:24:26.038+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:24:26.041+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:24:26.066+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.050 seconds
[2023-11-16T12:24:56.416+0000] {processor.py:157} INFO - Started process (PID=576) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:24:56.418+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:24:56.421+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:24:56.419+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:24:56.432+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:24:56.429+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:24:56.432+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:24:56.461+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.054 seconds
[2023-11-16T12:25:01.519+0000] {processor.py:157} INFO - Started process (PID=577) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:25:01.521+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:25:01.522+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:25:01.522+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:25:01.528+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:25:01.527+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:25:01.528+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:25:01.556+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.048 seconds
[2023-11-16T12:25:31.850+0000] {processor.py:157} INFO - Started process (PID=585) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:25:31.852+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:25:31.854+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:25:31.853+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:25:31.862+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:25:31.860+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dags.py", line 29
    docker_url='unix://var/run/docker.socks',
    ^
SyntaxError: invalid syntax
[2023-11-16T12:25:31.863+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:25:31.899+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.055 seconds
[2023-11-16T12:25:42.568+0000] {processor.py:157} INFO - Started process (PID=586) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:25:42.570+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:25:42.571+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:25:42.570+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:25:42.656+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:25:42.654+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T12:25:42.672+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:25:42.994+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:25:42.994+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:25:43.069+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.506 seconds
[2023-11-16T12:26:18.779+0000] {processor.py:157} INFO - Started process (PID=599) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:26:18.838+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:26:19.071+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:26:18.925+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:26:23.225+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:26:23.173+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T12:26:23.635+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:28:55.478+0000] {processor.py:157} INFO - Started process (PID=615) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:28:55.531+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:28:55.947+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:28:55.563+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:29:25.728+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:29:25.381+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T12:29:26.076+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:30:17.642+0000] {processor.py:157} INFO - Started process (PID=628) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:30:17.663+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:30:17.681+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:30:17.675+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:30:18.357+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:30:18.318+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T12:30:18.507+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:30:53.908+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:30:53.826+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:31:54.681+0000] {processor.py:157} INFO - Started process (PID=29) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:31:54.685+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:31:54.688+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:31:54.688+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:31:54.703+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:31:54.701+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T12:31:54.711+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:31:54.892+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:31:54.891+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:31:54.937+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.261 seconds
[2023-11-16T12:32:25.893+0000] {processor.py:157} INFO - Started process (PID=37) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:32:25.895+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:32:25.911+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:32:25.909+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:32:25.985+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:32:25.979+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T12:32:26.010+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:32:26.116+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:32:26.116+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:32:26.197+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.326 seconds
[2023-11-16T12:33:01.829+0000] {processor.py:157} INFO - Started process (PID=45) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:33:01.912+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:33:02.260+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:33:02.218+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:33:02.867+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:33:02.821+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T12:33:02.942+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:33:03.298+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:33:03.295+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:33:03.472+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 1.933 seconds
[2023-11-16T12:33:33.949+0000] {processor.py:157} INFO - Started process (PID=53) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:33:33.951+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:33:33.956+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:33:33.955+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:33:33.984+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:33:33.981+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T12:33:33.993+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:33:34.041+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:33:34.041+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:33:34.075+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.142 seconds
[2023-11-16T12:34:04.513+0000] {processor.py:157} INFO - Started process (PID=61) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:34:04.515+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:34:04.534+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:34:04.531+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:34:04.650+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:34:04.642+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T12:34:04.669+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:34:04.839+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:34:04.839+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:34:04.956+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.471 seconds
[2023-11-16T12:34:49.963+0000] {processor.py:157} INFO - Started process (PID=68) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:34:49.995+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:34:50.095+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:34:50.081+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:34:50.422+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:34:50.384+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T12:34:50.519+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:35:21.576+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:35:21.569+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:36:08.373+0000] {processor.py:157} INFO - Started process (PID=76) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:36:08.375+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:36:08.380+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:36:08.378+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:36:08.472+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:36:08.464+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T12:36:08.492+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:36:08.844+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:36:08.844+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:36:08.911+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.561 seconds
[2023-11-16T12:36:42.076+0000] {processor.py:157} INFO - Started process (PID=89) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:36:42.269+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:36:42.379+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:36:42.293+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:36:47.350+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:36:46.979+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T12:36:48.107+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:37:30.977+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:37:30.917+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:38:14.877+0000] {processor.py:157} INFO - Started process (PID=102) to work on /opt/airflow/dags/dags.py
[2023-11-16T12:38:15.024+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T12:38:15.125+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:38:15.066+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T12:38:22.120+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:38:21.821+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T12:38:24.480+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T12:38:35.136+0000] {logging_mixin.py:154} INFO - [2023-11-16T12:38:35.117+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T12:38:35.760+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 21.286 seconds
[2023-11-16T13:01:09.681+0000] {processor.py:157} INFO - Started process (PID=30) to work on /opt/airflow/dags/dags.py
[2023-11-16T13:01:09.685+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T13:01:09.688+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:09.687+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T13:01:09.745+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:09.739+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T13:01:09.764+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T13:01:10.023+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:10.023+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T13:01:10.486+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.810 seconds
[2023-11-16T13:01:19.572+0000] {processor.py:157} INFO - Started process (PID=31) to work on /opt/airflow/dags/dags.py
[2023-11-16T13:01:19.574+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T13:01:19.588+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:19.584+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T13:01:19.751+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:19.737+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T13:01:19.800+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T13:01:20.021+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:20.021+0000] {taskinstance.py:1939} ERROR - {'DAG Id': 'Backend_run', 'Task Id': 'run_docker_task', 'Run Id': 'scheduled__2023-01-05T00:00:00+00:00', 'Hostname': 'b631f0e58258', 'External Executor Id': 'd271b04d-471f-41e4-b4e1-ae4db7f70232'}
[2023-11-16T13:01:20.133+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:20.133+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=Backend_run, task_id=run_docker_task, execution_date=20230105T000000, start_date=20231116T123844, end_date=20231116T130120
[2023-11-16T13:01:20.196+0000] {processor.py:790} INFO - Executed failure callback for <TaskInstance: Backend_run.run_docker_task scheduled__2023-01-05T00:00:00+00:00 [failed]> in state failed
[2023-11-16T13:01:20.271+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:20.270+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T13:01:20.341+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:20.340+0000] {dag.py:3722} INFO - Setting next_dagrun for Backend_run to 2023-01-26T00:00:00+00:00, run_after=2023-01-27T00:00:00+00:00
[2023-11-16T13:01:20.398+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.863 seconds
[2023-11-16T13:01:29.567+0000] {processor.py:157} INFO - Started process (PID=38) to work on /opt/airflow/dags/dags.py
[2023-11-16T13:01:29.569+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T13:01:29.573+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:29.573+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T13:01:29.601+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:29.599+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T13:01:29.615+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T13:01:29.652+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:29.652+0000] {taskinstance.py:1939} ERROR - {'DAG Id': 'Backend_run', 'Task Id': 'run_docker_task', 'Run Id': 'scheduled__2023-01-25T00:00:00+00:00', 'Hostname': 'b631f0e58258', 'External Executor Id': '98b7d87d-b9ea-4aac-bda3-c40c0a2a4e28'}
[2023-11-16T13:01:29.671+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:29.671+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=Backend_run, task_id=run_docker_task, execution_date=20230125T000000, start_date=20231116T123845, end_date=20231116T130129
[2023-11-16T13:01:29.682+0000] {processor.py:790} INFO - Executed failure callback for <TaskInstance: Backend_run.run_docker_task scheduled__2023-01-25T00:00:00+00:00 [failed]> in state failed
[2023-11-16T13:01:29.684+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:29.684+0000] {taskinstance.py:1939} ERROR - {'DAG Id': 'Backend_run', 'Task Id': 'run_docker_task', 'Run Id': 'scheduled__2023-01-25T00:00:00+00:00', 'Hostname': 'b631f0e58258', 'External Executor Id': '98b7d87d-b9ea-4aac-bda3-c40c0a2a4e28'}
[2023-11-16T13:01:29.690+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:29.690+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=Backend_run, task_id=run_docker_task, execution_date=20230125T000000, start_date=20231116T123845, end_date=20231116T130129
[2023-11-16T13:01:29.692+0000] {processor.py:790} INFO - Executed failure callback for <TaskInstance: Backend_run.run_docker_task scheduled__2023-01-25T00:00:00+00:00 [failed]> in state failed
[2023-11-16T13:01:29.696+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:29.696+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-16T13:01:29.713+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:01:29.713+0000] {dag.py:3722} INFO - Setting next_dagrun for Backend_run to 2023-01-27T00:00:00+00:00, run_after=2023-01-28T00:00:00+00:00
[2023-11-16T13:01:29.727+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.166 seconds
[2023-11-16T13:02:28.618+0000] {processor.py:157} INFO - Started process (PID=39) to work on /opt/airflow/dags/dags.py
[2023-11-16T13:02:28.672+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T13:02:28.839+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:02:28.826+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T13:02:34.066+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:02:32.815+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T13:02:52.253+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T13:03:50.765+0000] {processor.py:157} INFO - Started process (PID=54) to work on /opt/airflow/dags/dags.py
[2023-11-16T13:03:50.776+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T13:03:50.866+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:03:50.852+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T13:03:51.365+0000] {logging_mixin.py:154} INFO - [2023-11-16T13:03:51.324+0000] {templater.py:76} ERROR - Failed to resolve template field 'command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 74, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sh ./run.sh
[2023-11-16T13:03:51.501+0000] {processor.py:839} INFO - DAG(s) dict_keys(['Backend_run']) retrieved from /opt/airflow/dags/dags.py
[2023-11-16T18:33:41.192+0000] {processor.py:157} INFO - Started process (PID=29) to work on /opt/airflow/dags/dags.py
[2023-11-16T18:33:41.194+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/dags.py for tasks to queue
[2023-11-16T18:33:41.201+0000] {logging_mixin.py:154} INFO - [2023-11-16T18:33:41.199+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/dags.py
[2023-11-16T18:33:41.207+0000] {logging_mixin.py:154} INFO - [2023-11-16T18:33:41.207+0000] {dagbag.py:323} INFO - File /opt/airflow/dags/dags.py assumed to contain no DAGs. Skipping.
[2023-11-16T18:33:41.207+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/dags.py
[2023-11-16T18:33:41.293+0000] {logging_mixin.py:154} INFO - [2023-11-16T18:33:41.292+0000] {taskinstance.py:1939} ERROR - {'DAG Id': 'Backend_run', 'Task Id': 'run_docker_task', 'Run Id': 'scheduled__2023-01-31T00:00:00+00:00', 'Hostname': '9ae86e27dab2', 'External Executor Id': '500e4815-f6db-4a25-8da0-a3aa2e8a6098'}
[2023-11-16T18:33:41.308+0000] {logging_mixin.py:154} INFO - [2023-11-16T18:33:41.308+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=Backend_run, task_id=run_docker_task, execution_date=20230131T000000, start_date=20231116T130317, end_date=20231116T183341
[2023-11-16T18:33:41.365+0000] {processor.py:790} INFO - Executed failure callback for <TaskInstance: Backend_run.run_docker_task scheduled__2023-01-31T00:00:00+00:00 [up_for_retry]> in state up_for_retry
[2023-11-16T18:33:41.367+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/dags.py took 0.182 seconds
